apis/eventstream/index.yaml

```diff
@@ -34,12 +33,30 @@ get:
             type: string
             description: https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#Event_stream_format
           examples:
-            head:
-              description: The node has finished processing, resulting in a new head
+            state:
+              description: |
+                The node has updated its state.  This event will be sent in three situations:
+
+                - a node has received a block, and updated state accordingly
+                - a node has _not_ received a block four seconds after the start of the slot, and updated state accordingly
+                - a node has carried out a chain reorganization, and updated state accordingly
+
+                The fields returned in the data for this event are:
+
+                  - **slot** the current slot as understood by the node
+                  - **block** the block root of the chain head as understood by the node
+                  - **state** the state root as understood by the node
+                  - **distance** the number of slots between the slot of the chain head and the current slot as understood by the node
```

Comment:
Is this essentially the number of empty slots after the head block?

Reply:
Yes.  I'm using "chain head" to mean the same thing as "head block"; can change the wording if you think it's unclear.

Reply:
I think chain head is fine here as long as we're consistent in that usage.  We may need a glossary at some point to define some key terms in one place.